app/views/teachers/_form.html.erb

```diff
@@ -10,12 +10,42 @@
       </ul>
     </div>
   <% end %>
+  <div class="outer-container">
+    <div class="container">
```

Comment:
Styling definitely smells. Would rather handle this later. Still don't have a clear picture of the finished product. This would be a good time to check with the "client", eh?